### PR TITLE
LPS-172436 fix Injection vulnerability in HttpInvoker via filename

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi Tina,

Resend from https://github.com/liferay-core-infra/liferay-portal/pull/3177, add the fix in client_http_invoker.ftl
This is for ticket https://issues.liferay.com/browse/LPS-172436.
Thanks for checking!

### CODE REVIEW

**Comment by Kvale1733** [comment]
@jeffwu0724 No regressions.

File: modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/client_http_invoker.ftl [hunk 1]
```diff
@@ -263,7 +263,7 @@ public class HttpInvoker {
 		fileName = fileName.replaceAll("\"", "");
 		fileName = fileName.replaceAll("\n", "");
 		fileName = fileName.replaceAll("\r", "");
-	
+
```
Comment (tinatian) [inline modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/client_http_invoker.ftl:266]: @jeffwu0724 This change does not belong to this commit, please merge it into your previous commit.

**Comment by jeffwu0724** [comment]
@tinatian sorry for the typo. I have move that change to the previous commit. Didn't modify the code at all, so the previous test should be good enough.

**Comment by tinatian** [comment]
ci:test:sf

**Comment by tinatian** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/129798
